lib/config/presets/local/index.ts

```diff
@@ -25,12 +36,9 @@ export function getPreset({
   if (!platform) {
     throw new Error(`Missing platform config for local preset.`);
   }
-  const resolver = resolvers[platform.toLowerCase() as PlatformId];
+  const resolver = resolvers[platform];
   if (!resolver) {
-    throw new Error(
-      // TODO: can be undefined? #7154
-      `Unsupported platform '${platform}' for local preset.`
-    );
+    throw new Error(`Unsupported platform '${platform}' for local preset.`);
```

Comment:
The current error message feels a bit clunky, how about this?

> The platform you're using ($platform) does not support local  presets.

Reply:
@rarkins WDYT?

Reply:
Sounds good

Reply:
```suggestion
    throw new Error(`The platform you're using ($platform) does not support local presets.`);
```